config/ember-try.js

```diff
@@ -77,6 +78,8 @@ module.exports = async function () {
           },
         },
       },
+      embroiderSafe(),
```

Comment:
Similarly to how by default ember-cli [adds this to travisci config](https://github.com/ember-cli/ember-addon-output/compare/v3.25.3...v3.26.1#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R57), we'll want to add this in our githubci config [here](https://github.com/smile-io/ember-polaris/blob/master/.github/workflows/ci.yml#L76)